smoke-test/src/test/java/org/opennms/smoketest/BSMAdminIT.java

```diff
@@ -28,70 +28,49 @@
 
 package org.opennms.smoketest;
 
-import java.io.IOException;
-import java.util.UUID;
-
-import org.apache.http.client.ClientProtocolException;
-import org.apache.http.client.methods.CloseableHttpResponse;
-import org.apache.http.client.methods.HttpPost;
-import org.apache.http.entity.StringEntity;
 import org.junit.Test;
-import org.opennms.core.web.HttpClientWrapper;
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebElement;
-
-import com.google.common.net.HttpHeaders;
-import com.google.common.net.MediaType;
-
-import static org.junit.Assert.assertEquals;
 
 public class BSMAdminIT extends OpenNMSSeleniumTestCase {
 
-    @Test
-    public void canReadBusinessServices() throws ClientProtocolException, IOException {
-        // Deferring this check until we can delete entries via the UI
-//        // Verify that a message is presented when there are no business services defined
-//        m_driver.get(BASE_URL + "opennms/admin/bsm/index.jsp");
-//        findElementByXpath("//*[contains(text(), 'There are no business services defined.')]");
+    public static final String RENAMED_SERVICE_NAME = "renamed_service";
+    private final String BSM_LIST_URL = BASE_URL + "opennms/admin/bsm/index.jsp";
+    private final String BASIC_SERVICE_NAME = "BasicService";
+    private final String BASIC_SERVICE_NAME_TO_CHANGE = "BasicServiceToChange";
 
-        // Create a new Business Service via the REST API
-        String servicePrefix = UUID.randomUUID().toString();
-        createServiceWithPrefix(servicePrefix);
-
-        // Verify that the Business Service appears in the list
-        m_driver.get(BASE_URL + "opennms/admin/bsm/index.jsp");
-        findElementByXpath("//*[contains(text(), '" + servicePrefix + "-name')]");
-
-        // The Business Service's attributes should also be displayed
-        WebElement el = findElementByXpath("//*[contains(text(), '" + servicePrefix + "-key')]");
-        WebElement parent = el.findElement(By.xpath(".."));
-        assertEquals(servicePrefix + "-key: " + servicePrefix + "-value", parent.getText());
+    @Test
```

Comment:
These tests assume that they are executed in the order defined, but JUnit does not provide this guarantee.

We should either rework the logic or rename the tests and use the @FixMethodOrder annotation.